common/xrdp_client_info.h

```diff
@@ -143,7 +143,7 @@ struct xrdp_client_info
   int use_frame_acks;
   int max_unacknowledged_frame_count;
 
-  int disableSSLv3; /* 0 = no, 1 = yes */
+  long ssl_protocols;
```

Comment:
We are playing with fire here. This would change the structure that is used by xorgxrdp and X11rdp. The first cleanup we should do would be to lock down `struct xrdp_client_info` to whatever external dependencies expect, and more the rest to some other structure. Or at very least put a big comment there and write `NO CHANGES ABOVE THIS LINE`